### PR TITLE
Add test for wrong attribute import in tracks

### PR DESCRIPTION
**Pull request by yasakova-anastasia**
### Motivation and context

Add test for wrong attribute import in tracks #3229

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by nmanovic** [comment]
@yasakova-anastasia , need to resolve conflicts

**Comment by yasakova-anastasia** [comment]
@nmanovic , resolved conflicts

File: cvat/apps/dataset_manager/tests/assets/tasks.json [hunk 1]
```diff
@@ -97,5 +97,18 @@
         ]
       }
     ]
+  },
+  "many jobs": {
```
Comment (nmanovic) [inline cvat/apps/dataset_manager/tests/assets/tasks.json:101]: I don't understand for now why the task has many jobs?

File: cvat/apps/dataset_manager/tests/assets/annotations.json [hunk 2]
```diff
@@ -43,6 +43,49 @@
       }
     ]
   },
+  "CVAT for video 1.1 attributes in tracks": {
+    "version": 0,
```
Comment (nmanovic) [inline cvat/apps/dataset_manager/tests/assets/annotations.json:47]: Annotations don't follow the exact scenario which was described in the issue. Just want to check. Is it OK?

File: cvat/apps/dataset_manager/tests/assets/tasks.json (hunk 1 above)
Comment (nmanovic) [reply cvat/apps/dataset_manager/tests/assets/tasks.json:101]: Also it has 5 frames. But the annotations file has frame 8.

File: cvat/apps/dataset_manager/tests/test_rest_api_formats.py [hunk 3]
```diff
@@ -237,6 +237,52 @@ def test_api_v1_check_widerface_with_all_attributes(self):
...
+                    url = self._generate_url_upload_tasks_annotations(task_id, upload_format_name)
+                    with open(file_zip_name, 'rb') as binary_file:
+                        self._upload_file(url, binary_file, self.admin)
+
+                    # equals annotations
+                    task_ann = TaskAnnotation(task_id)
+                    task_ann.init_from_db()
+                    task_data = TaskData(task_ann.ir_data, Task.objects.get(pk=task_id))
+                    extractor = CvatTaskDataExtractor(task_data, include_images=include_images)
+                    data_from_task_after_upload = Dataset.from_extractors(extractor)
```
Comment (zhiltsov-max) [inline cvat/apps/dataset_manager/tests/test_rest_api_formats.py:264]: Can we extract this fragment into a function?

File: cvat/apps/dataset_manager/tests/assets/tasks.json (hunk 1 above)
Comment (yasakova-anastasia) [reply cvat/apps/dataset_manager/tests/assets/tasks.json:101]: The issue describes a task with several segments. In this test, we get 13 frames divided into 3 segments.

File: cvat/apps/dataset_manager/tests/assets/annotations.json (hunk 2 above)
Comment (yasakova-anastasia) [reply cvat/apps/dataset_manager/tests/assets/annotations.json:47]: These annotations also caught this bug. I changed it to the annotations from the issue description.

File: cvat/apps/dataset_manager/tests/test_rest_api_formats.py (hunk 3 above)
Comment (yasakova-anastasia) [reply cvat/apps/dataset_manager/tests/test_rest_api_formats.py]: Added function for this.

File: cvat/apps/dataset_manager/tests/assets/tasks.json (hunk 1 above)
Comment (nmanovic) [reply cvat/apps/dataset_manager/tests/assets/tasks.json:101]: OK. Now I see.

**Review by nmanovic — APPROVED** [review]

**Pull request merged**